File: database/src/services/userXEnterprise.js
const Enterprise = require('./../infra/models/enterprise');
const User = require('./../infra/models/user');
const Transaction = require('./../infra/models/transaction');
const UserXEnterprise = require('./../infra/models/userXEnterprise');
const BaseService = require('./../infra/services/baseService');

const includeModels = [
    {
        model: User,
        as: 'user'
    },
    {
        model: Enterprise,
        as: 'enterprise'
    },
    {
        model: Transaction,
        as: 'transactions'
    }
];


class UserXEnterpriseService extends BaseService {}
const service = new UserXEnterpriseService(UserXEnterprise, includeModels);
module.exports = service;

File: backend/app/services/userXEnterprise.py
from typing import Any, Optional, Union

from app.infra.services.base_service import BaseService
from app.schemas.userXEnterprise import CreateUserXEnterprise, UpdateUserXEnterprise

from app.config import settings

class UserXEnterpriseService(BaseService[CreateUserXEnterprise, UpdateUserXEnterprise]):
    pass

user_x_enterprise_service = UserXEnterpriseService(url=f"{settings.PRUEBA_DATABASE}/user-x-enterprises")